package-build/build-package.sh

```diff
@@ -0,0 +1,30 @@
+#!/bin/bash
+
+SRC_DIR=$1
+
+LIB='libcpr'
+
+LIB_DIR="$LIB-$VERSION"
+DEBIAN_DIR="$LIB_DIR/debian"
+
+ARCHIVE_NAME="$LIB_DIR.tar.gz"
+
+echo -e "Preparing tar archive and directory\n"
+cp -r $SRC_DIR $LIB_DIR
+
+tar --exclude-vcs -czf $ARCHIVE_NAME $LIB_DIR
+tar -xzf $ARCHIVE_NAME
+
+cd $LIB_DIR
+
+echo -e "\n\nCalling debmake\n"
+debmake -e 'philip.saendig@gmail.com' -f 'Philip Saendig'
```

Comment:
In case you plan to continue maintaining the packages build for cpr, we can keep it that way. But then I would suggest I create you for example `philip.saendig@libcpr.org` or more generic `deb@libcpr.org` as mail address.
Else, you can change the maintainer to me.

Reply:
I'm planning to continue maintaining the package. However, I think that using `deb@libcpr.org` is the best option. I updated the CI scripts so that `deb@libcpr.org` is now used as the maintainer E-Mail

Reply:
I can setup a forward rule, so you also get a mail once someone sends one to `deb@libcpr.org`. Are you interested in that?

Reply:
Yes, I'm interested. Thank you